### DISCUSSION TITLE
鲍勃无法替换 映射导致天梯卡门

### DISCUSSION THREAD

**Discussion by catshitz** [Pik-4/HsMod#204] [Ideas (许愿池)]
### Please check before submitting an issue/提交问题前请检查

- [ ] I have searched the issues and haven't found anything relevant/我搜索了这些问题，没有找到任何相关信息
- [x] I have read the README and haven't found anything relevant/我认真阅读了README，没有找到任何相关信息
- [x] I have tried repairing Hearthstone and reinstalling HsMod according to the ReadMe, but the problem persists/我尝试过修复炉石，并按照ReadMe重新安装HsMod，但是问题依然存在。
- [ ] I will upload LogOutput.log, HsMod.cfg, HsSkins.cfg /我将上传LogOutput.log, HsMod.cfg, HsSkins.cfg
- [x] I know how to reproduce the issue which may not be specific to my device/ 我知道如何重现该问题，但该问题可能并非特定于我的设备

### Describe the bug/描述你的问题。

[https://github.com/Pik-4/HsMod/issues/101#issue-2590367284](url)
和这个问题一致

鲍勃换肤会导致天梯匹配卡死，第一种匹配转盘无限滚动，无法进入对局，第二种进去一直卡在开局界面，头像刚到下方位置，牌库为疲劳状态，候选牌无法加载，界面持续处于该状态，不能进行游戏，只能手动认输并硬退出游戏重开。

默认鲍勃57110，只要选了就会自动一并选上偏好117409和117425，无法单选；单选自己有的一个104477，不选默认鲍勃，一样是上述问题。

等于现在酒保鲍勃的皮肤根本就不能替换。

### To Reproduce/如何复现

任意替换战旗鲍勃头像皮肤，如：

```
# 鲍勃
57110:102916,82062,104477,98784
```

### Expected behavior/期望结果

鲍勃成功换肤

**Comment by Pik-4** [comment]
hsskins只支持英雄替换。

**Comment by freechao123** [comment]
我也出现了，棋盘好像也不行

**Comment by WkField** [comment]
https://github.com/Pik-4/HsMod/issues/60
